UKHomeOffice/nmsw-ui
# NMSW-384 Your details skeleton page

# Ticket

NMSW-384

----

## AC

No functionality but can put together page layout & render tests

----

## To test

- Run app
- Go to localhost:3000
- Sign in
- Click on `Your details` in the nav
- > The design should match the link to the prototype in the ticket
- Click on 'Change your details'
- > You should be taken to a page with the h1 'Change your details'
- Click on 'Change your password'
- > You should be taken to a page with the h1 'Change your password'

----

## Notes

The delete your account button will have no functionality in this ticket
